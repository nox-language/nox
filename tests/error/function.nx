fun main() = (
    var value = func 0;
)
