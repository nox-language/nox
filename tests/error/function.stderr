error: Undefined function `func`
   --> tests/error/function.nx:2:17
  |
2 |     var value = func 0;
  |                 ^^^^

